subsys/bluetooth/controller/Kconfig

```diff
@@ -121,38 +121,39 @@ config BT_CTLR_SUBRATING_SUPPORT
 config BT_CTLR_CHANNEL_SOUNDING_SUPPORT
 	bool
 
-config BT_CTLR
-	bool "Bluetooth Controller"
-	help
-	  Enables support for SoC native controller implementations.
-
-if BT_CTLR
+# Virtual option that all local LL implemenations should select
+config HAS_BT_CTLR
+	bool
```

Comment:
Is it correct to say, that we want to let HAS_BT_CTLR have the default value
```
DT_CHOSEN_ZEPHYR_HCI := zephyr,zephyr,bt-hci

config HAS_BT_CTLR
	bool
	default $(dt_chosen_enabled,$(DT_CHOSEN_ZEPHYR_HCI))
```
If that is the case, then I would assume that we don't need `select HAS_BT_CTLR` in any place. We could even consider getting rid of `config HAS_BT_CTLR` completely.

Reply:
The DT chosen is used for *all* HCI drivers, whereas `BT_CTLR`/`HAS_BT_CTLR` is specifically for situations where you have a local link layer, so the above doesn't really work.

Reply:
Thanks, you are right